### DISCUSSION TITLE
Question about Run-time for Whisper

### DISCUSSION THREAD

**Discussion by maxwithrice** [openai/whisper#1034] [Show and tell]
Hello!

When I first started using Whisper it followed the time estimates that I see many people mentioning in various threads, about 1-3 minutes per 15 minutes of audio. 

However, within the last day or two, this run-time has extended to multiple hours. I'm including the information in terminal as well as the file information. Anyone have any guesses on why this may have occurred or what I could do to fix it?

<img width="265" alt="file_info_livingston_github" src="https://user-images.githubusercontent.com/124815187/223218498-a4e392b4-9f89-4b13-8c6d-2c33a7f65cc5.png">
<img width="581" alt="terminal_livingston_github" src="https://user-images.githubusercontent.com/124815187/223218501-a3711317-3376-4bb1-a2bf-6e0c2a59d5f8.png">

**Comment by jwnacnud** [comment]
Have you tried rebooting?

**Reply by maxwithrice** [reply]
I rebooted and that resolved the issue :). Thank you!!

**Comment by jongwook** [comment]
Hi, the discrepancy you're seeing is likely the inference speed difference on GPU vs CPU. It appears that you're running whisper on CPU (the screenshot says `FP16 is not supported on CPU; using FP32 instead`), and the 1-3 minutes for 15-min audio is likely measured on a capable NVIDIA GPU.

You can install a CUDA-enabled pytorch if you have a Windows or Linux machine with an NVIDIA GPU, or use Google Colab which provides a GPU runtime. You can also venture with [ROCm](https://github.com/openai/whisper/discussions/55) on AMD GPUs or [MPS](https://github.com/openai/whisper/pull/382) on ARM-based Macs, but these appear to be more error-prone.
